runtime/j9vm/javanextvmi.cpp

```diff
@@ -254,11 +253,23 @@ JVM_VirtualThreadMountBegin(JNIEnv *env, jobject thread, jboolean firstMount)
 	J9JavaVM *vm = currentThread->javaVM;
 	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
 
+	Trc_SC_VirtualThreadMountBegin_Entry(currentThread, thread, firstMount);
+
 	vmFuncs->internalEnterVMFromJNI(currentThread);
 	f_monitorEnter(vm->liveVirtualThreadListMutex);
 	j9object_t threadObj = J9_JNI_UNWRAP_REFERENCE(thread);
 
-	assert(IS_JAVA_LANG_VIRTUALTHREAD(currentThread, threadObj));
+	Assert_SC_true(IS_JAVA_LANG_VIRTUALTHREAD(currentThread, threadObj));
+
```

Comment:
I think you can test if the tracepoint is enabled before loading the continuation with `if (TrcEnabled_[TracePointName]) {`

Reply:
Updated.